src/openrct2/platform/platform.h

```diff
@@ -35,6 +35,10 @@
 #define MAX_PATH 260
 #endif
 
+#ifndef MAX_COMMAND_LENGTH
+#define MAX_COMMAND_LENGTH 512
```

Comment:
I don't think allocating that big a buffer would be sane. But 512 seems a bit wrong too, especially in case where `PATH_MAX` is defined to 4096.

How about `2*MAX_PATH`?